types/prettier/index.d.ts

```diff
@@ -97,9 +97,16 @@ export interface RequiredOptions extends doc.printer.Options {
      * @default true
      */
     bracketSpacing: boolean;
+    /**
+     * Put the > of a multi-line HTML (HTML, JSX, Vue, Angular) element at the end of the last line instead of being
```

Comment:
```suggestion
     * Put the `>` of a multi-line HTML (HTML, JSX, Vue, Angular) element at the end of the last line instead of being
```

Reply:
I've applied that change. Thanks for the feedback @Shinigami92 ;).

Regarding the prettier format, I thought the same thing when I saw the result but due that it's the standard format for the repo I added those changes.